src/js/constants/deviceBreakpoints.ts

```diff
@@ -0,0 +1,4 @@
+export const DeviceBreakpoints = {
```

Comment:
since this is a constant, i think it should be uppercase snake-case

Reply:
although if bento routes are also named like this we can keep it for consistency